sylvia-derive/src/parser.rs

```diff
@@ -181,6 +181,7 @@ pub struct ContractMessageAttr {
     pub exec_generic_params: Vec<Path>,
     pub query_generic_params: Vec<Path>,
     pub variant: Ident,
+    pub is_custom_msg: bool,
```

Comment:
It is not `is`. It may be `has`, but in general, bool fields in structures always looks suspicious to me, as they could be just calculated inline or like some content is missing. Here I am wondering if keeping the `path` of the `content` or at least its span may be useful for error reporting, but maybe not - unless we use it we may leave it this way. Bot not with this name.